nilearn/image/tests/test_image.py

```diff
@@ -283,12 +299,14 @@ def test_mean_img():
     img1 = nibabel.Nifti1Image(data1, affine=affine)
     img2 = nibabel.Nifti1Image(data2, affine=affine)
     img3 = nibabel.Nifti1Image(data3, affine=affine)
-    for imgs in ([img1, ],
-                   [img1, img2],
-                   [img2, img1, img2],
-                   [img3, img1, img2],  # Mixture of 4D and 3D images
-                  ):
-
+    for imgs in (
+        [
+            img1,
```

Comment:
I find this weird

Reply:
yup it is one those weird oddities with black (see below on a dummy file) where I usually bite the bullet and stop worrying and learn to enjoy the autoformatting.


```bash
$ black --diff tmp.py

--- tmp.py      2023-03-03 09:05:50.161572 +0000
+++ tmp.py      2023-03-03 09:05:51.257681 +0000
@@ -1,7 +1,9 @@
 for imgs in (
-    [img1,],
+    [
+        img1,
+    ],
     [img1, img2],
     [img2, img1, img2],
     [img3, img1, img2],  # Mixture of 4D and 3D images
 ):
     print(imgs)
would reformat tmp.py
```

Reply:
removing the comma allows to prevent this behavior